SECURITY.md

```diff
@@ -78,13 +87,12 @@ Fortunately, between go-mod-boostrap and [microservice self seeding](https://git
 sudo bash
 
 cd cmd/core-metadata
-SecretStore_TokenFile=/tmp/edgex/secrets/edgex-core-metadata/secrets-token.json Logging_EnableRemote="false" ./core-metadata
+SecretStore_TokenFile=/tmp/edgex/secrets/edgex-core-metadata/secrets-token.json ./core-metadata
```

Comment:
Setting the TokenFile no longer needed as this is the default is the TOML files. Have to also set `EDGEX_SECURITY_SECRET_STORE=true`  so this is the command-line I use:
```
EDGEX_SECURITY_SECRET_STORE=true ./core-metadata
```

Reply:
Changed as suggested with additional note that this setting is optional.